### DISCUSSION TITLE
should the organization have an team@epiverse-trace.org email?

### DISCUSSION THREAD

**Discussion by avallecam** [epiverse-trace/epiverse-trace.github.io#123] [General]
This could be used in any direct contact email account managed by the person currently in place. 

Examples of this requirement are in the:
- [collaboration paths](https://github.com/epiverse-trace/epiverse-trace.github.io/blob/b398a9a73100eab323f01ff7f6d3b8169952f8a5/index.qmd#L24)
- [tutorials](https://github.com/epiverse-trace/tutorials/blob/de3f3691db0ed441aa6cae29e034f579f786684c/CONTRIBUTING.md?plain=1#L107)

To solve this, I'm adding my institutional email, but this may be an edit to use in more than one place, and it may be challenging to find all after a while unless mapped elsewhere.

**Comment by adamkucharski** [comment]
Yes, that could make sense, and would avoid having to make repeat edits to documentation. Have registered epiverse-trace.org as a placeholder.
